spec/controllers/posts_controller_spec.rb

```diff
@@ -639,6 +650,33 @@
       end
     end
 
+    context "mark hidden" do
+      it "marks hidden" do
+        post = create(:post)
+        user = create(:user)
+        login_as(user)
+        expect(post.ignored_by?(user)).not_to be_true
+
+        put :update, id: post.id, hidden: 'true'
+        expect(response).to redirect_to(post_url(post))
+        expect(flash[:success]).to eq("Post has been hidden")
+        expect(post.reload.ignored_by?(user)).to be_true
```

Comment:
Can you also check that this doesn't muck with the read_at timestamp? I've seen that happen because of how I overloaded the views.